src/hooks/useWalletENSAvatar.ts

```diff
@@ -1,19 +1,15 @@
-import { useCallback } from 'react';
-import { useDispatch } from 'react-redux';
 import { PROFILES, useExperimentalFlag } from '@/config';
-import { useWallets } from '@/hooks';
-import { getWalletENSAvatars } from '@/redux/wallets';
+import { useWalletsStore } from '@/redux/wallets';
+import { useCallback } from 'react';
 
 export default function useWalletENSAvatar() {
-  const dispatch = useDispatch();
+  const walletsStore = useWalletsStore();
```

Comment:
fyi when using these hooks, this will subscribe this hook to any change in the store. Instead, select only what you need:
```
const refreshWalletENSAvatars = useWalletsStore(state => state.refreshWalletENSAvatars);
```